docs/guides/bigquery_and_gcp_sync_with_cloud.md

```diff
@@ -5,18 +5,18 @@ parent: Guides
 permalink: /docs/guide/bigquery
 ---
 
-# Synchronising Google Bigquery for the first time
+# Synchronizing Google Bigquery for the first time
 
-In this guide we'll walk you through an example of how to connect Raito Cloud to your BigQuery data warehouse through the Raito CLI. We'll 
+In this guide, we'll walk you through an example of how to connect Raito Cloud to your BigQuery data warehouse through the Raito CLI. We'll 
 - make sure that Raito CLI is installed and available
 - log into Raito Cloud and create a data source
 - create a service account and assign the correct IAM Roles
 - configure Raito CLI to connect to Raito Cloud and synchronize with the previously-created data source
 - run a first sync
   
-For this guide you will need access to Raito Cloud and you also need access to GCP and optionally GSuite
-If you don't have the latter, you can request a 30-day free trial. 
-If you don't have access to Raito Cloud, [request a trial](https://www.raito.io/trial){:target="_blank"}. 
+For this guide, you will need access to Raito Cloud and you also need access to GCP and optionally GSuite
+If you don't have the former, you can request a 30-day free trial. 
```

Comment:
I think 'the latter' is really meant and correct here; a 30 day trial for GCP. The next point mentions the Raito trial. It would be best to order the points differently though (e.g. 'the former' before 'the latter').